lib/swig/generate_swig_config.py

```diff
@@ -0,0 +1,538 @@
+import sys
```

Comment:
What does this file do? It needs a lot of commenting!

Reply:
it generates interface files for SWIG. Some interfaces require many templates specializations and the interfaces for models look alike. So this file does all the copy-paste work :smiley: Additionally it reads the models files and creates template specializations for them, so when a new model is added to GeNN, it will wrap it without any additional work.

Reply:
It would also be really nice if this stuff generated pydoc help - especially as someone who doesn't know how it works ``help(pygenn.NNmodel.addNeuronPopulation)`` gives me no clues as to what parameters are

Reply:
actually everything is supposed to work via GeNNModel class

Reply:
that explains why **that** wasn't working then :smile: